<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreHotelDialog</class>
 <widget class="QDialog" name="ScoreHotelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>感谢您使用本程序预订酒店!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border images:url(:/images/images/star.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 13pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>请为酒店打分！:D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
